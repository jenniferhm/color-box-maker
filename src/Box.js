import React from 'react';

class Box extends React.Component {
  render() {
    console.log(this.props);
    const boxStyle = {
      height: `${this.props.height}px`,
      width: `${this.props.width}px`, 
      backgroundColor: this.props.color
      }
    return (
    <div style={boxStyle}>
      <button onClick={(evt => this.props.removeBox(this.props.id, evt))}>X</button>
    </div>)
  }
}

export default Box;