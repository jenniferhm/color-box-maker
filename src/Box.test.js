import React from "react";
import { shallow } from "enzyme";
import Box from './Box';


describe('<Box /> rendering', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Box height="200px" width="200px" color="blue" />);
  });

  describe('blue box', () => {
    it('renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});