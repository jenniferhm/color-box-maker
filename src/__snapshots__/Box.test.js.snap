// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> rendering blue box renders correctly 1`] = `ShallowWrapper {}`;
